src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -43,13 +43,17 @@ const CustomFieldInputSingleChoice = forwardRef(function CustomFieldInputSingleC
   const clear = () => {
     setValue(undefined);
     setSearchValue(undefined);
+    inputRef.current.focus();
   };
 
   const clearIcon = () => {
     if (!props.readOnly && (value || searchValue)) {
       return (<Icon
         name={iconClear.id}
         onClick={clear}
+        onEnter={clear}
+        ariaLabel={'Remove selected choice'}
```

Comment:
Should this say
```suggestion
        ariaLabel={'Clear selected choice'}
```
To keep the nomenclature consistent?